src/main/java/com/bina/varsim/util/Position.java

```diff
@@ -0,0 +1,44 @@
+package com.bina.varsim.util;
+
+/**
+ * Created by bayo on 2/28/15.
+ * base class for base-0 or base-1 location
+ * in C++ this can be done with template<char val> class Position; I'm not so sure about java
```

Comment:
there's a better template-way of doing this in C++ to check at compile time, but I can't figure how to do it in java